src/ErrorHandler/HtmlRenderer.php

```diff
@@ -54,9 +66,14 @@ private function htmlEncode(string $text): string
 
     private function renderTemplate(string $template, array $params): string
     {
-        $path = __DIR__ . '/templates/' . $template . '.php';
+        if (in_array($template, $this->defaultTemplates)) {
+            $path = __DIR__ . '/templates/' . $template . '.php';
```

Comment:
Can this path finding be moved into params directly?
This would allow us to get rid of the "default templates"

Reply:
Idk if i understand correctly - did you mean to move building default directory path in params? Because inside HtmlRenderer there are functions (i.e. renderCallstackItem) that call renderTemplate only passing template filename so we have to somehow create full path inside HtmlRenderer. 

I can move building default path to params.php so the above line would look sth like:
`$path = $this->defaultTemplatePath . '/' . $template . '.php';` where $this->defaultTemplatePath would be set with with value passed from params in construct.

Reply:
Good idea, i didn't like the fact that sometimes there was template name passed and sometimes template path to renderTemplate